MVC/controller.js

```diff
@@ -7,5 +7,6 @@ exports. getTopics = (req, res, next) => {
     })
 }
 
-
-// console.log('controller.js bonjour');
\ No newline at end of file
+exports. getAPI = (req, res, next) => {
+    res.status(200).send(fetchAPI())
```

Comment:
this looks a little weird but it's only because you don't technically need a model for this ticket because it's not interacting with the database. You could just require them in here and send them (remember anything from the controller needs to be send on a meaningful key)